pkg/summarizer/summary.go

```diff
@@ -668,7 +668,7 @@ func staleHours(tab *configpb.DashboardTab) time.Duration {
 }
 
 // updateTab reads the latest grid state for the tab and summarizes it.
-func updateTab(ctx context.Context, tab *configpb.DashboardTab, group *configpb.TestGroup, groupReader gridReader, allowFuzzyFlakiness bool) (*summarypb.DashboardTabSummary, error) {
+func updateTab(ctx context.Context, tab *configpb.DashboardTab, group *configpb.TestGroup, groupReader gridReader, allowFuzzyFlakiness, allowIgnoredColumns, allowMinNumberOfRuns bool) (*summarypb.DashboardTabSummary, error) {
```

Comment:
I dunno that this is your responsibility, but since a lot of similar parameters/return values are getting passed around I think we might want to consider a struct to hold them, for less confusion down the road. (If you want to add a TODO to that effect, that would be awesome!)

The return from gridMetrics(), and passing all the boolean options around, are the main things I'm looking at.

Reply:
that's a great idea! I went ahead and aggregated the feature flags and column/cell metrics into respective structs. LMK what you think. Thanks!